src/components/MathInput.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="math-input">
-    <input type="text" v-model="mathInput" size="100" /> <br />
+    <input type="text" v-model="mathInput" size="100" v-bind="$attrs" /> <br />
```

Comment:
Does this in combination with inheritAttrs lead to us being able to pass aria-label attributes and so on down to the input element? If so that is really nice.
We should still think about adding a label to all input fields, or atleast make it optional so we can transition in the future

Reply:
Yes, that's exactly what it does. 
And yes, we should probably add a label to every input field.